docs/source/README.md

```diff
@@ -288,35 +263,11 @@ The Source will provide output information about readiness or errors via the
     consist of a non-empty sequence of letters, digits, hyphen, underscore,
     period, or colon.
 
-    2. (Deprecated) Each exchange belongs to one of a set of exchange kinds/types implemented by
-    the server. The exchange types define the functionality of the exchange - i.e.
-    how messages are routed through it. Once an exchange is declared, its type
-    cannot be changed.  The common types are "direct", "fanout", "topic" and
-    "headers".
-
-    3. (Deprecated) Durable and Non-Auto-Deleted exchanges will survive server restarts and remain
-    declared when there are no remaining bindings.  This is the best lifetime for
-    long-lived exchange configurations like stable routes and default exchanges.
-
-    4. (Deprecated) Non-Durable and Auto-Deleted exchanges will be deleted when there are no
-    remaining bindings and not restored on server restart.  This lifetime is
-    useful for temporary topologies that should not pollute the virtual host on
-    failure or after the consumers have completed.
-
-    5. (Deprecated) Non-Durable and Non-Auto-deleted exchanges will remain as long as the server is
-    running including when there are no remaining bindings.  This is useful for
-    temporary topologies that may have long delays between bindings.
-
-    6. (Deprecated) Durable and Auto-Deleted exchanges will survive server restarts and will be
-    removed before and after server restarts when there are no remaining bindings.
-    These exchanges are useful for robust temporary topologies or when you require
-    binding durable queues to auto-deleted exchanges.
-
-    7. Note: RabbitMQ declares the default exchange types like 'amq.fanout' as
+    2. Note: RabbitMQ declares the default exchange types like 'amq.fanout' as
     durable, so queues that bind to these pre-declared exchanges must also be
     durable.
 
-    8. Optional amqp.Table of arguments that are specific to the server's implementation of
+    3. Optional amqp.Table of arguments that are specific to the server's implementation of
```

Comment:
Is this relevant still? I don't think there's a field we have for this

Reply:
true, removed